docs/CODE_GUIDELINES.md

```diff
@@ -366,43 +366,45 @@ static void test();
 **[back to top](#table-of-contents)**
 
 ## 7. Headers
-Included header files have to be sorted alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.
+Included header files have to be sorted (case sensitive) alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.
 
 Header order has to be:
 * Own header file
-* Other Kodi includes
+* Other Kodi includes (platform independent)
+* Other Kodi includes (platform specific)
 * C and C++ system files
 * Other libraries' header files
+* special Kodi headers (PlatformDefs.h, system.h and system_gl.h)
```

Comment:
Why are those "special" and why shall they placed after all other headers?

Reply:
PlatformDefs.h and system.h declares many windows like structs and defines which shouldn't pollute other headers if not needed
system_gl.h is a `Other libraries' header files` wrapper

Reply:
okay, thanks. makes sense.